coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CalciteTrinoUDFOperatorTransformers.java

```diff
@@ -0,0 +1,220 @@
+    operatorTransformers.add(createOperatorTransformer(SqlStdOperatorTable.SUBSTRING, 3, "SUBSTR"));
+    operatorTransformers.add(createOperatorTransformer(HiveRLikeOperator.RLIKE, 2, "REGEXP_LIKE"));
+    operatorTransformers.add(createOperatorTransformer(HiveRLikeOperator.REGEXP, 2, "REGEXP_LIKE"));
+
+    // JSON Functions
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("get_json_object"), 2, "json_extract"));
+
+    // map various hive functions
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("pmod"), 2, "mod",
+        "[{\"op\":\"+\",\"operands\":[{\"op\":\"%\",\"operands\":[{\"input\":1},{\"input\":2}]},{\"input\":2}]},{\"input\":2}]",
+        null));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("base64"), 1, "to_base64"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("unbase64"), 1, "from_base64"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("hex"), 1, "to_hex"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("unhex"), 1, "from_hex"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("array_contains"), 2, "contains"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("regexp_extract"), 3, "regexp_extract",
+        "[{\"input\": 1}, {\"op\": \"hive_pattern_to_trino\", \"operands\":[{\"input\": 2}]}, {\"input\": 3}]", null));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("instr"), 2, "strpos"));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("decode"), 2,
+        "[{\"regex\":\"(?i)('utf-8')\", \"input\":2, \"name\":\"from_utf8\"}]", "[{\"input\":1}]", null));
+
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("to_date"), 1, "date",
+        "[{\"op\": \"timestamp\", \"operands\":[{\"input\": 1}]}]", null));
+    operatorTransformers
+        .add(
+            createOperatorTransformer(hiveToCalciteOp("date_add"), 2, "date_add",
+                "[{\"value\": 'day'}, {\"input\": 2},  "
+                    + "{\"op\": \"date\", \"operands\":[{\"op\": \"timestamp\", \"operands\":[{\"input\": 1}]}]}]",
+                null));
+    operatorTransformers
+        .add(
+            createOperatorTransformer(hiveToCalciteOp("date_sub"), 2, "date_add",
+                "[{\"value\": 'day'}, " + "{\"op\": \"*\", \"operands\":[{\"input\": 2}, {\"value\": -1}]}, "
+                    + "{\"op\": \"date\", \"operands\":[{\"op\": \"timestamp\", \"operands\":[{\"input\": 1}]}]}]",
+                null));
+    operatorTransformers.add(createOperatorTransformer(hiveToCalciteOp("datediff"), 2, "date_diff",
+        "[{\"value\": 'day'}, {\"op\": \"date\", \"operands\":[{\"op\": \"timestamp\", \"operands\":[{\"input\": 2}]}]}, "
+            + "{\"op\": \"date\", \"operands\":[{\"op\": \"timestamp\", \"operands\":[{\"input\": 1}]}]}]",
+        null));
+
+    // DALI functions
```

Comment:
Let's modify all `dali functions` to be `LinkedIn specific UDFs` in this class? Would be great if we void mentioning the team name everywhere.

Reply:
Good point, done.